lib/views/options.html

```diff
@@ -157,13 +157,13 @@
                   </p>
                   <small>
                     <span class="label label-danger">Note:</span>
-                    <span style = "color: rgba(75, 174, 79, 1.00);">Green, </span>
-                    <span style = "color: rgba(164, 59, 64, 1.00);">Mexican Red</span>
-                    and <span style = "color: rgba(243, 66, 53, 1.00);">Red</span>
+                    <span style = "color: rgba(86, 179, 90, 1.00);">#4BAE4F, </span>
```

Comment:
Will change this in the next PR

On Thursday 9 June 2016, Bruno <notifications@github.com> wrote:

> In lib/views/options.html
> <https://github.com/OiWorld/MindTheWord/pull/130#discussion_r66428874>:
>
> > @@ -157,13 +157,13 @@
> >                    </p>
> >                    <small>
> >                      <span class="label label-danger">Note:</span>
> > -                    <span style = "color: rgba(75, 174, 79, 1.00);">Green, </span>
> > -                    <span style = "color: rgba(164, 59, 64, 1.00);">Mexican Red</span>
> > -                    and <span style = "color: rgba(243, 66, 53, 1.00);">Red</span>
> > +                    <span style = "color: rgba(86, 179, 90, 1.00);">#4BAE4F, </span>
>
> I find it a bit strange that the usdr is supposed to insert colors in
> rgba, but here we refer to the colors by their hexadecimal numbers... Maybe
> we could uniformize this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OiWorld/MindTheWord/pull/130/files/3fb3b11906e22e2072dd1da4a782396152124d23#r66428874>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACVrys9YseuOJ-o-XsWtHreZOIezUg5Aks5qKAPZgaJpZM4Ixm12>
> .
>


-- 
Regards,
Rohan Katyal

Please ignore the typos and brevity, email sent using a small form factor
device